cs3api4lab/api/cs3_share_api.py

```diff
@@ -60,9 +60,11 @@ def create(self, endpoint, file_path, grantee, idp, role, grantee_type):
             self.log.info(create_response)
             return self._map_given_share(create_response.share)
         if create_response.status.code == cs3_code.CODE_INTERNAL:
```

Comment:
I would go with 'else' clause here, because there are more error codes that implicate failure than CODE_INTERNAL, besides CODE_INTERNAL does not occur only when the share already exists but for other reasons as well